tutoraspects/templates/aspects/apps/superset/pythonpath/collect_dbt_lineage.py

```diff
@@ -0,0 +1,195 @@
+"""
```

Comment:
A lot of this file is adapted from here: https://github.com/slidoapp/dbt-superset-lineage/blob/main/dbt_superset_lineage/pull_dashboards.py#L264

I thought I had documented that here, and will do so if this seems like a good direction.